src/Dapr.Jobs/Dapr.Jobs.csproj

```diff
@@ -0,0 +1,40 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+    <PropertyGroup>
+        <TargetFrameworks>net6;net8</TargetFrameworks>
+        <ImplicitUsings>enable</ImplicitUsings>
+        <Nullable>enable</Nullable>
+        <PackageId>Dapr.Jobs</PackageId>
+        <Title>Dapr Jobs Authoring SDK</Title>
+        <Description>Dapr Jobs SDK for scheduling jobs and tasks with Dapr</Description>
+        <VersionPrefix>0.1.0</VersionPrefix>
```

Comment:
Would we want to version the "sub" SDKs independently?  While I think it's *technically* the right thing to do, as the number of NuGet packages increase, version management becomes more of a challenge and there's an increase risk of customer confusion (e.g. why do I reference 1.14 of the Dapr.Client library but 0.1.0 of the Dapr.Jobs library).  Traditionally we've followed the version of the Dapr runtime, at least for major/minor versions, and the build automation infrastructure assumes packages are all published as a set.  We could change all that, of course, but I'm not sure if there's significant benefit.